src/appleseed.python/CMakeLists.txt

```diff
@@ -213,6 +213,11 @@ target_link_libraries (appleseed.python
     appleseed
     ${Boost_PYTHON_LIBRARY}             # all Boost libraries but Boost.Python3
     ${PYTHON_LIBRARIES}
+    ${APPLESEED_DEPS_STAGE_DIR}/oiio-release/lib/OpenImageIO.lib
```

Comment:
That's Windows-specific and won't compile on other platforms (as reported by Travis).

Use `link_against_oiio (appleseed.python)` instead. Check how it's done in the case of appleseed's core library.

Reply:
I already did, just hadn't pushed it yet.